code/modules/organs/robolimbs_custom.dm

```diff
@@ -0,0 +1,219 @@
+/datum/robolimb
+	var/includes_tail			//Cyberlimbs dmi includes a tail sprite to wear.
+	var/includes_wing			//Cyberlimbs dmi includes a wing sprite to wear.
+	var/list/whitelisted_to		//List of ckeys that are allowed to pick this in charsetup.
+
+//////////////// For-specific-character fluff ones ///////////////// May be viable to place these into a custom_item subfolder, in order to allow CI Repo integration.
+// arokha : Aronai Sieyes
+/datum/robolimb/kitsuhana
+	company = "Kitsuhana"
+	desc = "This limb seems rather vulpine and fuzzy, with realistic-feeling flesh."
+	icon = 'icons/mob/human_races/cyberlimbs/_fluff_vr/aronai.dmi'
+	blood_color = "#5dd4fc"
+	includes_tail = 1
+	lifelike = 1
+	unavailable_to_build = 1
+	suggested_species = "Tajara"
+	whitelisted_to = list("arokha")
```

Comment:
Same request as above, could you please remove whitelisted stuff from downstream?